### DISCUSSION TITLE
Pointer pressed doesn`t work

### DISCUSSION THREAD

**Discussion by kochkaqqq** [AvaloniaUI/Avalonia#14994] [Q&A]
![image](https://github.com/AvaloniaUI/Avalonia/assets/134213884/dd452646-a5e6-4069-9ef0-0541bc865016)
![image](https://github.com/AvaloniaUI/Avalonia/assets/134213884/43cf6f30-bae3-49eb-8102-42077724fbb3)
When I click on border, Border_PointerPressed_3 doesn`t call. And if I add TextBlock (or another control) inside of the border  event will happens only if click on text. I want the event happens when I click on any point of border.

**Comment by thevortexcloud** [comment]
You may need to tunnel the event.

https://docs.avaloniaui.net/docs/get-started/wpf/tunnelling-events

**Comment by jp2masa** [comment]
This has been asked many times. For example, https://github.com/AvaloniaUI/Avalonia/discussions/9794#discussioncomment-4507418.
